samples/wifi/promiscuous/prj.conf

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# WPA supplicant
+CONFIG_WPA_SUPP=y
+
+# Promiscuous mode
+CONFIG_NET_PROMISCUOUS_MODE=y
+CONFIG_NRF700X_PROMISC_DATA_RX=y
+CONFIG_NRF_WIFI_LOW_POWER=n
```

Comment:
any specific reason for this?

Reply:
Well, this is a common problem for the `promiscuous mode` and not this sample, so, either driver or FW should handle this.User shouldn't worry about powersave or power management esp. if that is hindering functionality. @udaynordic  @VivekUppunda WDYT?

Reply:
@krish2718 - the reason the LOW power was set to `NO`was because it was informed that certain pieces of code in the firmware only come into being when this is disabled. Hence, the need for this currently.

Reply:
Can you please share some details? Nonetheless the point I am making was is that applicable only for this sample and doesn't apply for promiscuous mode in general (assume customer application)?

Reply:
ok, so, it still looks like it's applicable to the feature not a sample, so, this should be done at the driver level.

Reply:
No...It has to be handled in the Firmware level and not the driver level if needed. As which portions of code that is susceptible to this Flag is only known to the Firmware team. When we were testing the feature - we were told to use the following build command from the UMAC team

west build -b nrf7002dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG=overlay-promiscuous-mode.conf -DCONFIG_NRF_WIFI_LOW_POWER=n

Reply:
Sure, that is what I was proposing earlier. But handling at FW level is complicated given that it's patch and we need feedback as to why PS fails, at driver level we can enforce it via Kconfig. (For all dependencies)

>  so, either driver or FW should handle this

Reply:
As it's going to be handled in FW/Driver, removing the config.